onnx_tf/backend_tf_module.py

```diff
@@ -0,0 +1,37 @@
+import tensorflow as tf
+from onnx_tf.pb_wrapper import OnnxNode
+
+class BackendTFModule(tf.Module):
+
+  def __init__(self, handlers, opset, strict, graph_def, backend):
+    super(BackendTFModule, self).__init__()
+    self.handlers = handlers
+    self.opset = opset
+    self.strict = strict
+    self.graph_def = graph_def
+    self.backend = backend
+
+  @tf.function
+  def __call__(self, **kwargs):
+    tensor_dict = kwargs
+
+    if self.graph_def.initializer:
+      input_dict_items = self.backend._onnx_initializer_to_input_dict_items(
+          self.graph_def.initializer)
+      initialized = {init.name for init in self.graph_def.initializer}
+    else:
+      input_dict_items = []
+      initialized = set()
+
+    tensor_dict.update(input_dict_items)
+
+    output = []
+    for node in self.graph_def.node:
+      onnx_node = OnnxNode(node)
+      output_ops = self.backend._onnx_node_to_tensorflow_op(
+          onnx_node, tensor_dict, self.handlers, opset=self.opset, strict=self.strict)
+      curr_node_output_map = dict(zip(onnx_node.outputs, output_ops))
+      tensor_dict.update(curr_node_output_map)
+
+    outputs = [tensor_dict[output] for output in self.outputs]
```

Comment:
Shouldn't there be a class variable for outputs? The use of self.outputs seems strange. Do you assume outputs variable is added by TF with tf.function?